docs/index.rst

```diff
@@ -697,7 +697,7 @@ The security in place between |sr| and the end clients, e.g. ``appSA``, is as fo
    .. sourcecode:: bash
 
       # First get the KAFKA_CLUSTER_ID
-      KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v1/metadata/id | jq -r ".id")
+     KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v3/clusters | jq ".data[0].cluster_id"")
```

Comment:
@rnpridgeon cp-demo code continues to refer to the `v1/metadata/id` endpoint instead of `v3/clusters` endpoint.

1. For consistency, it should all be the same.  Need to decide on one or the other.
2. Which one should it be in 6.0.x?  What's the difference between the two endpoints, when should a user hit one versus the other?

Reply:
ERP is on by default, MDS isn't. So I'd personally use the new one. I thought I got them all but alas I guess I missed some.

Reply:
1. `http://localhost:8091/v1/metadata/id` is also used in the scripts themselves, so those would need to change as well.
2. What are the other consequences of changing the endpoint?  What if the embedded RP dies?

Reply:
1. okay I'm impartial but if we want to demonstrate the new API we should use the new one
2. It goes away; this is true for the former UI if MDS dies though so using the former implementation isn't more robust.

Reply:
Ok, in that case, please update all instances (see scripts):

```
scripts/validate/validate_rest_proxy.sh:KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v1/metadata/id | jq -r ".id")
scripts/validate/validate_bindings.sh:KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v1/metadata/id | jq -r ".id")
scripts/helper/functions.sh:  KAFKA_CLUSTER_ID=$(curl -s http://kafka1:8091/v1/metadata/id | jq -r ".id")
```